### PR TITLE
Dep bump

### PR DESCRIPTION
**Pull request by landonreed**
Bumped out-of-date dependencies and removed unused ones.

This includes Leaflet 0.7 -> 1.0.3, which included some work on ControlPoint to make sure dragging works OK.

There was also some work to make sure the GridLayers are working properly.

Additionally (sorry for the bloat):

- removal of a few unused files (ManagerNavbar, DatatoolsNavbar)
- fix for MTC to request feeds after third party sync

### CODE REVIEW

**Comment by evansiroky** [comment]
let's merge the lint fixes into this branch and then review

File: lib/editor/actions/map/index.js [hunk 1]
```diff
@@ -23,6 +23,50 @@ export function addControlPoint (controlPoint, index) {
   }
 }
 
+export function dragControlPointEnded (index, controlPoint, evt, pattern) {
+  return {
+    type: 'HANDLE_CONTROL_POINT_DRAG_END',
```
Comment (evansiroky) [inline lib/editor/actions/map/index.js]: Is there a reducer that handles this action?

Comment (landonreed) [inline lib/editor/actions/map/index.js]: No, it's all really handled below in `handleControlPointDragEnd`.

File: lib/manager/components/NotesViewerPanel.js [hunk 2]
```diff
@@ -33,16 +32,29 @@ export default class NotesViewerPanel extends Component {
...
-      }}>
+      <h3
+        onClick={this._toggleExpanded}
+        onKeyDown={(e) => {
+          if (document.activeElement === e.target && e.which === 13) {
+            this._toggleExpanded()
+          }
+        }}
+        role='button'
+        tabIndex={0}
```
Comment (evansiroky) [inline lib/manager/components/NotesViewerPanel.js]: We should come up with a convention about what to use for clickable items.  Either an `a` or `button` within this h3 would be more appropriate and then we wouldn't need the onKeyDown handler.

File: lib/editor/actions/map/index.js (hunk 1 above)
Comment (evansiroky) [inline lib/editor/actions/map/index.js]: Let's get rid of these unused actions.

File: lib/manager/components/NotesViewerPanel.js (hunk 2 above)
Comment (landonreed) [inline lib/manager/components/NotesViewerPanel.js]: Oh boy, NotesViewerPanel is old code... isn't even imported anywhere.  Deleting.

**Pull request merged**